app/controllers/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  def show
+    @created_events = current_user.created_events
+
+    @future_events = current_user.attended_events.where('event_date > ?', Date.today).order('event_date ASC')
+    @past_events = current_user.attended_events.where('event_date <  ?', Date.today).order('event_date DESC')
```

Comment:
- [ ] [MANDATORY] As the previous reviewer suggested, take advantage of `past` and `future` scopes:
  Example:       
  ```ruby
    @past_events = current_user.attended_events.past
    @future_events = current_user.attended_events.future
  ```